src/solver/optimisation/opt_calcul_des_pmin_MUT_MDT.cpp

```diff
@@ -35,6 +35,8 @@
 #include <yuni/io/file.h>
 #include "opt_fonctions.h"
 
+#define ZERO_PMIN 1.e-2
```

Comment:
Is it possile to use const expression (const or constxpr) instead of define ?

Reply:
It should be, will look into it.

Reply:
Suggestion to rename headers to `*_constants.h` accordingly

Reply:
> Suggestion to rename headers to `*_constants.h` accordingly

[Done](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1146/commits/181a127d72744fa04cca71ee7e58af7b3fce9c4c)